pkg/services/profiles/update.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/google/uuid"
 )
 
-const UpdateCommitMessage = "Update Profile manifests"
+const UpdateCommitMessage = "Update profile manifests"
 
 type UpdateOptions struct {
```

Comment:
this looks identical to AddOptions struct as well

Reply:
good shout, I'll refactor into 1